modules/porous_flow/tests/buckley_leverett/bl01.i

```diff
@@ -114,7 +114,7 @@
     material_property = PorousFlow_viscosity
   [../]
   [./permeability]
-    type = PorousFlowMaterialPermeabilityConst
+    type = PorousFlowPermeabilityConst
```

Comment:
Great, when you've completely finished, try running the tests in debug mode

./run_tests -j 4 --dbg

and

./run_tests -j 4 --dbg --heavy

and that's all!  Then you can put in a PR to idaholab/moose (i can help if i'm at work).